GRANT SELECT (c1) ON t1 TO regressuser2;
NOTICE:  00000: test_event_trigger: ddl_command_start GRANT
ERROR:  42703: column "c1" of relation "t1" does not exist
